src/edu/ccsu/cs505/hw2/PayingParty.java

```diff
@@ -0,0 +1,14 @@
+package edu.ccsu.cs505.hw2;
+
+/**
+ * @author Julie Shevchenko
+ * PayingParty interface, which Vendor and Customer implement
+ * Has an abstract method pay which takes an amount as an argument and returns nothing
+ **/
```

Comment:
Note be sure Javadoc comment always begins with /** and ends with single */ If you have more asterisks for either the Javadoc generator will not pick up the Javadoc comment.